docs/constrained_spherical_deconvolution/single_tissue_csd.rst

```diff
@@ -22,7 +22,7 @@ Prerequisites
 Single-tissue CSD relies on *single-shell* high angular resolution diffusion
 imaging (HARDI) data, containing at least one non-zero *b*-value. Ideally, the
 *b*-value used should be in the region of 2,500 -- 3,000 s/mm² (for *in vivo*
-brain), although good results can still be obtained using more routine *b* =
+human brains), although good results can still be obtained using *b* =
```

Comment:
Why restrict this to _human_ brains only? Do we have any reason to believe that this wouldn't apply to other animals? I appreciate we don't have data to show that it is optimal in animals, but we certainly have no evidence or rationale to suggest that it wouldn't. I think this qualification is overly restrictive...

Reply:
It's the other way around: not trying to restrict it, just trying to limit the scope of that statement to the evidence we have.  Just playing it safe essentially.  I'm also not stating that is would *not* be ok / optimal in all or certain animals.  Furthermore putting in between () brackets to just carefully limit the confidence of the statement to that.

But most importantly: also *not* stating that it would *not* hold.  I think that's fair.  I'm not confident endorsing beyond that evidence... you know, that's what we did voice we were ok with as an approach to these kinds of things, right.  In this case, that evidence doesn't even exist with/for *any* of us; whereas, for example, I do have enough evidence to convince me of a certain level of accuracy statement about the `dhollander` algo, but still we hold back because that hasn't been offered in a digestible way to some others.

So well, I feel it's a safe and gentle added nuance between brackets that fits the level of confidence I'm comfortable endorsing.

Reply:
(I'm also not against adding references that add evidence for some animals; adding references to peer reviewed material is always fair game)